config/config.go

```diff
@@ -198,3 +198,96 @@ func (cfg *Cache) GetBaseURL() string {
 func (cfg *Configuration) GetCachedAssetURL(uuid string) string {
 	return fmt.Sprintf("%s/cache?%s", cfg.CacheURL.GetBaseURL(), strings.Replace(cfg.CacheURL.Query, "%PBS_CACHE_UUID%", uuid, 1))
 }
+
+// Set the default config values for the viper object we are using.
+func ConfigViper(v *viper.Viper) {
```

Comment:
opinion: `SetDefaults` might be a bit more clear?

Reply:
`SetupViper` also sounds good.

I'm just shooting for something which publicizes the fact that it actually _mutates_ the argument object. `Set`, `Setup`, `Init`, etc. do that. `Configure` isn't bad either... but `Config` is used as a noun too frequently.